app/controllers/hub/clients/organizations_controller.rb

```diff
@@ -11,6 +11,7 @@ class OrganizationsController < ApplicationController
       def edit;end
 
       def update
+        authorize!(:edit_organization, @client)
```

Comment:
If we already have permissions set up in `Ability` for both `Client` & `VitaPartner, then I believe we can do this check without having to create a custom action.

```ruby
      def update
        @client.update(vita_partner: @vita_partners.find(client_params[:vita_partner_id]))
        redirect_to hub_client_path(id: @client.id)
      end
```

Reply:
We're using a custom action to create a helper in the view that displays the button or hides it based on your role permissions.